Main.kt

```diff
@@ -0,0 +1,10 @@
+fun main() {

```

Comment:
Main is not required, but I can see why you created it, proving the solution via tests is the best way.